geoid/__init__.py

```diff
@@ -122,7 +124,7 @@
     'sdsec': 5,  # State-School District (Secondary)
     'sduni': 5,  # State-School District (Unified)
     'sldl': 3,  # State Legislative District Lower
-    'sldu': "3",  # State Legislative District Upper
```

Comment:
> On Sep 4, 2015, at 10:11 AM, Kazbek <notifications@github.com> wrote:
> 
> In geoid/__init__.py:
> 
> Ok. I was confused by the name of the dict. Looks like it contains lengths mapping. Also the comment "# Lengths in number of decimal digits" seems invalid.

Well, the values in the lengths dict are actually all lengths ( or widths), although it’s more correct to say the lengths are measured in characters than decimal digits. 

The “decimal digits” part distinguishes from the other measure of length, in terms of base-62 digits. The civick.Gvid class represents the numbers in base 62, which is shorter. It would also be sensible to have a base-36 version — for case independence — which would have yet another measure of length/width. 


eric. 

----------------------------------------------------------------------------------------------------------
Eric Busboom                                                                                (619) 363 2607        
eric@busboom.org                                                         http://www.busboom.org